include/ur_api.h

```diff
@@ -3396,7 +3396,7 @@ urUSMHostAlloc(
     ur_context_handle_t hContext,  ///< [in] handle of the context object
     const ur_usm_desc_t *pUSMDesc, ///< [in][optional] USM memory allocation descriptor
     ur_usm_pool_handle_t pool,     ///< [in][optional] Pointer to a pool created using urUSMPoolCreate
-    size_t size,                   ///< [in] size in bytes of the USM memory object to be allocated
+    size_t size,                   ///< [in] minumum size in bytes of the USM memory object to be allocated
```

Comment:
s/minumum/minimu/g

Here and in a bunch of places below.

Reply:
Fixed, Thanks for noting that!